src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodLengthCheck.java

```diff
@@ -73,26 +73,102 @@
  * &lt;module name="MethodLength"/&gt;
  * </pre>
  * <p>
- * To configure the check so that it accepts methods with at most 60 lines:
+ * Example:
+ * </p>
+ * <pre>
+ * public class MyClass {
+ *   public MyClass() {  // constructor (line 1)
+ *        /&#42; line 2
+ *            ...
+ *           line 150 &#42;/
+ *   } // line 151, violation, as it is over 150
+ *     // comments and empty lines are counted towards
+ *
+ *   public void firstExample() { // line 1
+ *       //  line 2
+ *       /&#42;  line 3
+ *           line 4 &#42;/
+ *   } // line 5, OK, as it is less than 150
+ *     // comments and empty lines are counted towards
+ *
+ *   public void secondExample() { // line 1
+ *       /&#42; line 2
+ *           ...
+ *          line 150 &#42;/
+ *   } // line 151, violation, as it is over 150
+ *     // comments and empty lines are counted towards
```

Comment:
there is no example with empty line

Reply:
I add the code example below for empty lines but after I ran `mvn clean verify`, I get `Unnecessary consecutive lines [noConsecutiveLines]` error.
```
public void thirdExample() { // line 1



} // line 5, OK, as it is less than 150
```

I would like to know whether if the example below is fine.

```
public void thirdExample() { // line 1

} // line 3, OK, as it is less than 150
```